npm i @themost/sqlite fails for nodejs v12

npm i @themost/sqlite fails for nodejs v12

```
../src/database.cc: In static member function ‘static void node_sqlite3::Database::Work_AfterLoadExtension(uv_work_t*)’:
../src/macros.h:91:13: error: no matching function for call to ‘v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)’
             ),                                                                 \
             ^
../src/database.cc:659:9: note: in expansion of macro ‘EXCEPTION’
         EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
         ^~~~~~~~~
In file included from ./.cache/node-gyp/12.16.1/include/node/node.h:63,
                 from ../node_modules/nan/nan.h:51,
                 from ../src/database.h:10,
                 from ../src/database.cc:4:
./.cache/node-gyp/12.16.1/include/node/v8.h:3065:24: note: candidate: ‘static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)’
   static Local<String> Concat(Isolate* isolate, Local<String> left,
                        ^~~~~~
./.cache/node-gyp/12.16.1/include/node/v8.h:3065:24: note:   candidate expects 3 arguments, 2 provided
```

--- a/modules/@themost/sqlite/package.json
+++ b/modules/@themost/sqlite/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@themost/sqlite",
-  "version": "2.2.4-dev.1",
+  "version": "2.2.4",
   "description": "MOST Web Framework 2.0 - SQLite data adapter",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",

--- a/modules/@themost/sqlite/package.json
+++ b/modules/@themost/sqlite/package.json
@@ -1,11 +1,10 @@
 {
   "name": "@themost/sqlite",
-  "version": "2.2.3",
+  "version": "2.2.4-dev.1",
   "description": "MOST Web Framework 2.0 - SQLite data adapter",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "scripts": {
-  },
+  "scripts": {},
   "peerDependencies": {
     "@themost/common": "^2.1.6",
     "@themost/query": "^2.1.3"
@@ -22,8 +21,8 @@
   "homepage": "https://github.com/themost-framework/themost-adapters#readme",
   "dependencies": {
     "async": "0.2.10",
-    "lodash": "^4.17.11",
-    "moment": "^2.22.2",
-    "sqlite3": "^4.0.4"
+    "lodash": "^4.17.15",
+    "moment": "^2.24.0",
+    "sqlite3": "^4.1.1"
   }
 }

